src/main/java/lotto/domain/LottoTickets.java

```diff
@@ -12,15 +12,29 @@
 
     public LottoTickets(final List<LottoTicket> lottoTickets) {
         Objects.requireNonNull(lottoTickets, NULL_ERROR_MESSAGE);
-        validateEmptyTickets(lottoTickets);
+        validateEmpty(lottoTickets);
         this.lottoTickets = lottoTickets;
     }
 
-    public List<LottoTicket> getLottoTickets() {
+    public LottoTickets merge(LottoTickets otherLottoTickets) {
+        LottoTickets mergedTickets = new LottoTickets(lottoTickets);
+        mergedTickets.addAll(otherLottoTickets);
+        return mergedTickets;
+    }
```

Comment:
```suggestion
    public LottoTickets merge(LottoTickets otherLottoTickets) {
        final List<LottoTicket> mergedTickets = new ArrayList<>(this.lottoTickets.size() + otherLottoTickets.lottoTickets.size());
        mergedTickets.addAll(this.lottoTickets);
        mergedTickets.addAll(this.otherLottoTickets.lottoTickets);
        return mergedTickets;
    }
```

기존의 값을 변화시키지 않고 사용해주는 것이 좋을 것 같네요!
객체를 불변하게 유지한다고 생각하시면 좋을 것 같습니다 :)

Reply:
LottoTickets mergedTickets = new LottoTickets(lottoTickets);
20번째 줄에서 LottoTickets.lottoTickets가 final List<LottoTicket>으로 정의가 되어있어서
재연링 코드랑 사실상 같은줄로 이해가 되는데요
혹시 재연링처럼 작성했을 때 다른 이점이 있을까요?